### ISSUE TITLE
Bronchial stenosis

### ISSUE THREAD

**Issue by nadja-ehmke** [obophenotype/human-phenotype-ontology#10857]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**
narrowing of the bronchial tubes, which branch off the trachea into the lungs

**Parent term (use hpo.jax.org/app)**
Abnormal bronchus morphology

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**
0000-0003-1449-9909

**Comment by pnrobinson** [comment]
@nadja-ehmke Hi!
This term is similar to [Bronchoconstriction HP:4000007](https://hpo.jax.org/browse/term/HP:4000007) but I think it means more of a permanent finding rather than the finding in Asthma etc? Can you provide details, what diseases are meant?

**Comment by pnrobinson** [comment]
Adding new term: 
- ID: HP:6001383
- label: Bronchial stenosis
- parents: [Abnormal bronchus morphology (HP:0025426)](https://hpo.jax.org/app/browse/term/HP:0025426)
- pmid: 
- def: A narrowing (constriction) of the lumen of a bronchus, which may involve one or more of the major airways branching from the trachea.
- comment: Bronchial stenosis may affect the main bronchi, lobar bronchi, or segmental bronchi. Clinical manifestations depend on the location and severity of narrowing and can include wheezing, dyspnea, recurrent pneumonia, or asymmetric lung ventilation on imaging.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10861
- synonyms: Bronchial narrowing(NONE)
